images/nginx-1.25/rootfs/Dockerfile

```diff
@@ -32,7 +32,8 @@ COPY --from=builder /usr/local /usr/local
 COPY --from=builder /opt /opt
 COPY --from=builder /etc/nginx /etc/nginx
 
-RUN apk update \
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories \ 
+  && apk update \
```

Comment:
What is the reason for adding the testing repo?

Packages in the testing repository will be considered "unstable".

Reply:
just a thought that alpine testing repo is including fixes, typically for ssl, so may get false positives

Reply:
Changed here to use testing only for OTEL packages, and we can monitor while that.

In parallel if we decide to not use it anymore we can follow @esigo idea's on having a separate image just with the libraries (grpc-dev, protobuf-dev, opentelemetry-cpp) that we need for the compilation